content/en/synthetics/guide/_index.md

```diff
@@ -18,6 +18,7 @@ private: true
 {{< /whatsnext >}}
 
 {{< whatsnext desc="Browser Tests:" >}}
+    {{< nextlink href="synthetics/guide/browser-tests-totp/" >}}Time-Based One Time Passwords (TOTPs) For Multi-Factor Authentication (MFA){{< /nextlink >}}
```

Comment:
Let's put this one under `Monitor an application that requires authentication` instead to keep some kind of order in the guide list.